examples/demo/canvas/cliptest.py

```diff
@@ -70,7 +67,7 @@ def moving_mouse_leave(self, event):
 
 
 class MainFrame(DemoFrame):
-    def _create_window(self):
+    def _create_component(self):
```

Comment:
`canvas.py` in the canvas examples directory also needs the same fix - after which i am running into a very interesting failure mode.

now that atleast one of the canvas examples is fixed, should we close #576 ?

Reply:
> canvas.py in the canvas examples directory also needs the same fix - after which i am running into a very interesting failure mode.

it looks like you got the canvas example running in #254 . it'll be good to fix it up in the master branch as wekk.

Reply:
I've closes #576.  I had it "working" just by making the same switch to use `_create_component`, but more will needed to truly fix up the whole example by rewriting it to avoid use of the custom multitouch support / at least make the functionality more useable with a mouse